src/styles/_variables.scss

```diff
@@ -0,0 +1,120 @@
+// Typography
+// ------------------------------------------------
+
+$font__default: "Satoshi Variable", sans-serif;
+$font__code: "PT Mono", monospace;
+
+// Colors
+// ------------------------------------------------
+
+// Brand colors
+$color__c4-button-blurple: rgba(113, 74, 235, 1);
+$color__c4-logo-blurple: #855eff;
+$color__sponsor-color: rgba(64, 203, 161, 1);
+
+// Black and white
+$color__white: rgba(255, 255, 255, 1);
+$color__black: rgba(2, 2, 3, 1);
+
+// Neutral palette
+$color__n-100: rgba(11, 10, 13, 1);
+$color__n-90: rgba(29, 27, 38, 1);
+$color__n-80: rgba(46, 43, 59, 1);
+$color__n-70: rgba(49, 46, 64, 1);
+$color__n-60: rgba(64, 61, 77, 1);
+$color__n-50: rgba(75, 71, 89, 1);
+$color__n-40: rgba(86, 82, 102, 1);
+$color__n-30: rgba(107, 102, 128, 1);
+$color__n-20: rgba(141, 138, 153, 1);
+$color__n-10: rgba(173, 171, 178, 1);
+$color__n-5: rgba(196, 194, 204, 1);
+$color__n-2: rgba(230, 230, 230, 1);
+
+// Spacing
+// ------------------------------------------------
+
+$spacing__base: 0.5rem; // 8px
+
+$spacing__xxs: calc(#{$spacing__base}/ 4); // 2px
```

Comment:
I was going to ask why these were left as `calc()` but it looks like SASS does [simplify them down](https://sass-lang.com/documentation/values/calculations) when it can so +1 on this syntax. (i.e. thought it might make the every browser redo the calcs but that's not the case.)

It also seems a bit suspicious to be bothering with rems if what you really mean is all the pixels in the comments, but I guess that's kind of the age-old CSS units debates and the idea is that it *could* scale without loss of generality…? https://css-tricks.com/is-it-better-to-use-ems-rems-than-px-for-font-size/

Reply:
Oh, actually by that same https://sass-lang.com/documentation/values/calculations link isn't it saying you should be using `calc($spacing__base / 4)` instead of `calc(#{$spacing__base}/ 4)`? So I think this might be something to change so that it **can** pre-calculate all these when it compiles?

Reply:
whaaaat since when can you use variables without interpolation inside sass's calc? TIL! That's great.